tests/e2e/e2e_test.go

```diff
@@ -53,14 +54,12 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 }, func(data []byte) {})
 
 var _ = Describe("Smoke", func() {
-	It("should create a tidb cluster", func() {
-		testCreate()
-	})
-	It("should upgrade a tidb cluster", func() {
-		testUpgrade()
-	})
-	It("should scale in/out a tidb cluster", func() {
-		testScale()
-	})
-
+	for i := 0; i < len(fixtures); i++ {
```

Comment:
```suggestion
	for _, fixture := range fixtures {
```

Reply:
This is a golang common mistake: https://github.com/golang/go/wiki/CommonMistakes
Should not use the `for ... range` style.

Reply:
Oh, I made this mistake too